ftw/referencewidget/converter.py

```diff
@@ -49,3 +50,27 @@ def toFieldValue(self, value):
     def toWidgetValue(self, value):
         if value:
             return '/'.join(value.getPhysicalPath())
+
+
+class ReferenceDataListWithChoiceConverter(converter.BaseDataConverter):
+    """Converter of IList of IRelationChoices."""
+
+    adapts(IList, IReferenceWidget)
+
+    def toFieldValue(self, value):
+        if not value:
+            return []
+        elif isinstance(value, (unicode, str)):
+            return [value]
+        else:
+            return map(
+                lambda path: path.lstrip(
```

Comment:
`lstrip` does not do what you expect it to do, see https://docs.python.org/2/library/string.html#string.lstrip
The parameter is `chars`, a list of chars which is stripped, not a specific text to remove from the left.

I think you want to do something like:
```python
site_path = '/'.join(api.portal.get().getPhysicalPath())
return map(lambda path: os.path.relpath(path, site_path), …)
```